javascript/javascript1/week1/week-1.js

```diff
@@ -0,0 +1,73 @@
+//* Age-ify (A future age calculator) *//
+
+const yearOfBirth = 2005;
+const yearFuture = 2076;
+
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//* Goodboy-Oldboy (A dog age calculator) *//
+
+const dogYearOfBirth = 2022;
+const dogYearFuture = 2034;
+
+const dogAge = dogYearFuture - dogYearOfBirth;
+shouldShowResultInDogYears = false;
+
+//* Acording to Google every human year equals approximately four or five years for a dog :) *//
+
+if (shouldShowResultInDogYears) {
+    console.log(`Your dog will be ${dogAge * 5} dog years old in ${dogYearFuture}.`);
+}
+
```

Comment:
Leaving spaces between the `if` and `else` branches is not necessary, you can just write: 

```javascript
if (condition) {
  // some code
} else {
  // some other code
}
```

It actually is easier to read this way. 🔍